roles/httpd/tasks/main.yml

```diff
@@ -45,11 +36,23 @@
 - name: Configure foreman-ssl vhost
   ansible.builtin.template:
     src: foreman-ssl-vhost.conf.j2
-    dest: /etc/httpd/conf.d/foreman-ssl.conf
+    dest: "{{ httpd_conf_dir }}/foreman-ssl.conf"
     mode: "0644"
 
-- name: Start Apache httpd
-  ansible.builtin.service:
+- name: Disable welcome page in container
+  ansible.builtin.file:
+    path: "{{ httpd_conf_dir }}/welcome.conf"
+    state: absent
+
+- name: Run Apache as a container
+  containers.podman.podman_container:
     name: httpd
+    image: "{{ httpd_container_image }}"
     state: started
-    enabled: true
+    volumes:
+      - "{{ httpd_ssl_dir }}:/etc/httpd/ssl"
+      - "{{ httpd_conf_dir }}:/etc/httpd/conf.d"
```

Comment:
Is this how it's intended to be used? Looking at https://catalog.redhat.com/software/containers/ubi9/httpd-24/61a60c3e3e9240fca360f74a this isn't how they think about it.